Testscripts/Linux/Linux-Test-Project-Tests.sh

```diff
@@ -34,6 +34,11 @@ ltp_git_url="https://github.com/linux-test-project/ltp.git"
 # The LTPROOT is used by some tests, e.g, block_dev test
 export LTPROOT="/opt/ltp"
 
+if [[ ! -z "$CUSTOM_LTP_ROOT" ]]; then
+	TOP_BUILDDIR="$CUSTOM_LTP_ROOT"
```

Comment:
what's the difference between $TOP_BUILDDIR and $LTPROOT?  Can we use a simple one and reduce the variables here? How can it be used in line 155 and line 139 with almost the same path?

Reply:
Per the comment in the file, some tests rely on $LTPROOT, but I agree we should rely on $TOP_BUILDDIR only and link LTPROOT to it. It looks both use the same path in the context.